Description: Upstream changes introduced in version 1:1.5.1-2ubuntu2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 python-numpy (1:1.5.1-2ubuntu2) oneiric; urgency=low
 .
   * numpy/distutils/fcompiler/gnu.py: add 'shared' to linker_so. LP: #829488.
 .
 The person named in the Author field signed this changelog entry.
Author: Matthias Klose <doko@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/829488

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- python-numpy-1.5.1.orig/numpy/distutils/fcompiler/gnu.py
+++ python-numpy-1.5.1/numpy/distutils/fcompiler/gnu.py
@@ -245,7 +245,7 @@ class Gnu95FCompiler(GnuFCompiler):
         'compiler_f90' : [None, "-Wall", "-fno-second-underscore"] + _EXTRAFLAGS,
         'compiler_fix' : [None, "-Wall", "-ffixed-form",
                           "-fno-second-underscore"] + _EXTRAFLAGS,
-        'linker_so'    : ["<F90>", "-Wall"],
+        'linker_so'    : ["<F90>", "-Wall", "-shared"],
         'archiver'     : ["ar", "-cr"],
         'ranlib'       : ["ranlib"],
         'linker_exe'   : [None, "-Wall"]
